packages/expo-env-info/src/index.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import path from 'path';
+
+import { actionAsync } from './diagnosticsAsync';
+
+const packageJson = () => require('../package.json');
+
+async function run() {
+  const args = process.argv.slice(2);
+
+  if (args.some(arg => ['-V', '-v', '--version'].includes(arg))) {
```

Comment:
Drop `-V` support since it is nonstandard outside of JS, a bit unfortunate that commander uses it.